spacelift/data_scheduled_delete_stack.go

```diff
@@ -0,0 +1,94 @@
+package spacelift
+
+import (
+	"context"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/spacelift-io/terraform-provider-spacelift/spacelift/internal"
+	"github.com/spacelift-io/terraform-provider-spacelift/spacelift/internal/structs"
+)
+
+func dataScheduledDeleteStack() *schema.Resource {
+	return &schema.Resource{
+		Description: "" +
+			"`spacelift_scheduled_delete_stack` represents a scheduling configuration " +
+			"for a Stack. It will trigger delete_task on the given timestamp.",
```

Comment:
What does `delete_task` mean in this context? Should it just be "It will trigger a stack deletion task" or something like that?

Reply:
you are right, adjusted description.